canary/generator/transformators/transformers_categorical.py

```diff
@@ -0,0 +1,94 @@
+from abc import ABC
+from copy import deepcopy
+
+import numpy as np
+
+from .base import Transformer
+
+
+class TransformerCategorical(Transformer, ABC):
+    def __init__(self):
+        super(TransformerCategorical, self).__init__(['categorical', 'enumerated',
+                                                      'boolean', 'flag'])
```

Comment:
Extra blank line? :)

Reply:
Where? I'm not sure 3 blank lines are ok in pep8

Reply:
Ouch, I saw 3 blank lines, but I was wrong. Sorry.